pkgs/applications/networking/instant-messengers/teams/default.nix

```diff
@@ -39,6 +39,13 @@ let
     license = licenses.unfree;
     maintainers = with maintainers; [ liff tricktron ];
     platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
Those CVEs are Linux-only?

Reply:
These are CVEs inherited from the outdated Electron Version (10.4.7, Chrome 85.something).
The Mac app is still maintained and at a much higher version.
(version History for Mac: https://learn.microsoft.com/en-us/officeupdates/teams-app-versioning)